add my allocation feature and accept and cancel button

shubhpatel232 opened the pull request.

shubhpatel232 committed:
add my allocation feature and accept and cancel button

shubhpatel232 commented:
@GuruBuilds please have a look.

GuruBuilds commented on SkillOnCall/booking/views.py:
```diff
@@ -68,3 +68,47 @@ def view_bookings(request):
+
+    return render(request, 'booking/my_allocation.html', {'bookings': bookings})
+
+@login_required	
+def confirm_booking(request, booking_id):
```
```suggestion
@login_required	
def confirm_booking(request, booking_id):
```

Hello @shubhpatel232 ,

I have an idea when provider cancal or confirm the booking we don't user the login_require decorator ?

What do think let me know so i can make changes in this code and try to make that possible

shubhpatel232 replied:
to allow this routes without login_require we need to introduce something like access token instead of direct using ID. so yes you can improve this in new PR.

GuruBuilds replied:
@shubhpatel232  okay its good idea to improve this in new PR. so i'll merge this PR

GuruBuilds commented on SkillOnCall/templates/base.html:
```diff
@@ -26,7 +26,12 @@
           <li class="nav-item">
             <a class="nav-link" href="{% url 'view_bookings' %}">My Bookings</a>
           </li>
```
we have to add condition here also `if user.is_authenticated`

shubhpatel232 committed:
make some UI imp

shubhpatel232 commented:
@GuruBuilds i have added one more commit to improve UI. it have some repeated code which can be reduced. i have just added it to show you one idea. you can push a commit for improving that.

Thank you.

GuruBuilds commented:
> @GuruBuilds i have added one more commit to improve UI. it have some repeated code which can be reduced. i have just added it to show you one idea. you can push a commit for improving that.
> 
> Thank you.

Hello @shubhpatel232 ,
Thanks to let me know i'll optimize the code in another pr as we discuss fir access_token
